packages/input-option/src/InputOption/InputOption.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {
+  storybookArgTypes,
+  // @ts-ignore
```

Comment:
why ts-ignore?

Reply:
was getting some build errors with the old lib version, but it's gone now